src/frontend/admin/src/services/repositories/auth/AuthRepository.ts

```diff
@@ -0,0 +1,13 @@
+import { checkStatus, fetchApi } from "@/services/http/HttpService";
+import { AuthenticatedUser } from "@/types/auth";
+
+export const authApiRoutes = {
+  me: `/users/me/`,
+};
+
+export class AuthRepository {
+  static async me(): Promise<AuthenticatedUser> {
+    const response = await fetchApi(authApiRoutes.me, { method: "GET" });
+    return checkStatus(response);
```

Comment:
to keep the same syntax as in other repositories
```suggestion
    return fetchApi(authApiRoutes.me, { method: "GET" }).then(checkStatus);
```